shared/sbp.js

```diff
@@ -17,6 +17,7 @@ const DOMAIN_SELECTOR_REGEX = /^([^/]+)(.+)/
 function sbp (path: string, ...data?: Array<*>) {
   var [, name, selector] = DOMAIN_SELECTOR_REGEX.exec(path)
   var domain = domains[name]
+  if (!domain) throw new Error(`Domain ${name} not Found`)
```

Comment:
as we discussed this has to be a fast-path and unneeded checks mustn't be here. This will get caught by an error thrown a few lines down below (and by any logging functions).

Reply:
Yes and that error is confusing and ambiguous when we know exactly what went wrong. I've gotten the error and its misleading.